go.mod

```diff
@@ -32,9 +33,9 @@ require (
 )
 
 require (
-	github.com/KarpelesLab/reflink v0.0.2 // indirect
 	github.com/Microsoft/go-winio v0.5.1 // indirect
 	github.com/Microsoft/hcsshim v0.9.3 // indirect
+	github.com/baiyubin/aliyun-sts-go-sdk v0.0.0-20180326062324-cfa1a18b161f // indirect
```

Comment:
Is this package still necessary after `go mod tidy`?

Reply:
Yes. This package seems to be imported by `github.com/aliyun/aliyun-oss-go-sdk`.